src/components/layout/navbar.module.scss

```diff
@@ -0,0 +1,57 @@
+.navbar {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 7rem;
+  border-right: 2px solid gray;
+  align-items: center;
+}
+
+.head {
+  width: 100%;
+  text-align: center;
+  padding-top: 1.5rem;
+}
+
```

Comment:
## [OPTIONAL]
- [ ] One thing to improve is the styling but everything is okay ✅